index.js

```diff
@@ -1,20 +1,33 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const app = express()
+const cors = require('cors')
+const app = express();
+
+app.use(cors());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 const Game = require('./src/game.js')
-
-app.get('/game',(req, res) => {
+const sequelize = require('./server/conf.js')
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
```

Comment:
refactor this as explained in the class to start the database before the api server